lm-cli/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This file keeps track of all notable changes to `License Manager CLI`.
 ## Unreleased
 
 
+## 4.2.3 -- 2025-01-16
```

Comment:
Need to add something here to indicate that the version was bumped to keep in sync with the other subprojects.